airbyte-webapp/build.gradle

```diff
@@ -46,8 +52,15 @@ task pnpmBuild(type: PnpmTask) {
 
     inputs.files commonConfigs
     inputs.file '.eslintrc.js'
+    inputs.file '.stylelintrc'
+    inputs.file 'scripts/calculate-source-hash.sh'
+    inputs.file 'scripts/load-declarative-schema.sh'
+    inputs.file 'orval.config.ts'
+    inputs.file 'vite.config.ts'
+    inputs.file 'index.html'
```

Comment:
Should we add `.env` to this list?

Reply:
`.env` is already part of that list via `commonConfigs` above.